pkg/manager/member/tikv_member_manager.go

```diff
@@ -638,6 +639,12 @@ func (m *tikvMemberManager) syncTidbClusterStatus(tc *v1alpha1.TidbCluster, set
 	} else if upgrading && tc.Status.PD.Phase != v1alpha1.UpgradePhase {
 		tc.Status.TiKV.Phase = v1alpha1.UpgradePhase
 	} else {
+		// If phase changes from UpgradePhase to NormalPhase, try to endEvictLeader for the last store.
```

Comment:
What if the phase changed from `upgrade` to `scale` directly?
I think we should move this out of this block.

Reply:
All right, I will change this. If upgrading == false and the phase changed to scaling, your advice can be helpful in this scenario.